values.yaml

```diff
@@ -355,7 +395,18 @@ st2garbagecollector:
   nodeSelector: {}
   tolerations: []
   affinity: {}
-
+  
+# Chatops service settings  
+st2chatops:
+  replicas: 1
```

Comment:
Let's remove replicas from configuration as st2chatops can't have more than 1 instances.
Adding a code comment about the fact that st2chatops doesn't scale would be helpful.